ref/Elementary_Drawing_Functions.md

```diff
@@ -266,3 +266,32 @@ The following piece of code shows a combined usage of the `drawcircle` and the `
 | ![Image](img/FillCircleX.png) |
 | ----------------------------- |
 | ****                          |
+
+------
+
+#### Drawing circular arc: `drawarc(‹point› A,<point> B, <point> C)`
```

Comment:
A good thing you thought about adding documentation! I might have forgotten it. Please use the angled quotation marks seen in the first argument also for the other two. This convention might one day be important if we e.g. try to automatically compute arity of documented functions, in order to add stable anchor names. The same holds for `fillarc`.